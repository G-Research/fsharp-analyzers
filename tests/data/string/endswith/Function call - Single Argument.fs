let f () = "f"
"g".EndsWith(f())
