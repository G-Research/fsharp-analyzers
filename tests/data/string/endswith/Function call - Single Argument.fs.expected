GRA-STRING-001 | Warning | (2,0 - 2,17) | The usage of String.EndsWith with a single string argument is discouraged. Signal your intention explicitly by calling an overload.
